src/smartContracts/smartContractTransactionsOutcomeParser.spec.ts

```diff
@@ -93,7 +93,7 @@ describe("test smart contract transactions outcome parser", () => {
         const transactionOnNetwork = new TransactionOnNetwork({
             nonce: 7n,
             function: "getUltimateAnswer",
-            smartContractResults: [new SmartContractResult({ data: Buffer.from("@6f6b@2a") })],
+            smartContractResults: [new SmartContractResult({ data: Buffer.from("QDZmNmJAMmE=", "base64") })],
```

Comment:
I think the inconsistencies between API and Proxy are not yet reconciled at the SDK level, thus a further fix is needed.

 - https://github.com/multiversx/mx-sdk-js-core/blob/main/src/core/transactionOnNetwork.ts#L71
 - https://github.com/multiversx/mx-sdk-js-core/blob/main/src/core/transactionOnNetwork.ts#L141

On proxy, it's an UTF-8 string (it's sub-components being hex-encoded):
 - https://devnet-gateway.multiversx.com/transaction/5647414afcd17d74ba0223837cefa47347946d806e1196d3e16558c881fc955d?withResults=true

On API, it's a base64-encoded string:
 - https://devnet-api.multiversx.com/transactions/5647414afcd17d74ba0223837cefa47347946d806e1196d3e16558c881fc955d


Good catch :tada: 

We'll discuss this a bit internally, then come back.

Reply:
Actually, I think we can approve this PR, since there's no change in the implementation with respect to this issue - there's only the fix  in `smartContractTransactionsOutcomeParser.ts`. And we can handle the Proxy - API inconsistency in a separate PR.